build-tools/packages/build-cli/src/handlers/checkFunctions.ts

```diff
@@ -405,38 +408,24 @@ export const checkPolicy: StateHandlerFunction = async (
     assert(context !== undefined, "Context is undefined.");
 
     if (shouldCheckPolicy === true) {
-        if (context.originalBranchName !== "main") {
+        if (!branchesRunDefaultPolicy.includes(context.originalBranchName)) {
             log.warning(
-                "WARNING: Policy check fixes are not expected outside of main branch!  Make sure you know what you are doing.",
+                `Policy check fixes are not expected on the ${context.originalBranchName} branch! Make sure you know what you are doing.`,
             );
         }
 
-        // await CheckPolicy.run([
-        //     "--fix",
-        //     "--exclusions",
-        //     path.join(
-        //         context.gitRepo.resolvedRoot,
-        //         "build-tools",
-        //         "packages",
-        //         "build-tools",
-        //         "data",
-        //         "exclusions.json"
-        //     )
-        // ]);
-
-        await exec(
-            `node ${path.join(
+        await CheckPolicy.run([
```

Comment:
```suggestion
        // oclif docs suggest using proper function calls into underlying APIs rather than calling a command directly
        // but it's very convenient in this case. check policy is a command where the CLI is the "best" interface; it doesn't
        // expose a better programmatic API and making one just to make it cleaner to call here seems unnecessary.
        await CheckPolicy.run([
```